web-components/src/components/cards/OrderSummaryCard/OrderSummaryCard.tsx

```diff
@@ -0,0 +1,163 @@
+import { ReactNode, useState } from 'react';
+import { EditButtonIcon } from 'web-components/src/components/buttons/EditButtonIcon';
+import { DenomIconWithCurrency } from 'web-components/src/components/DenomIconWithCurrency/DenomIconWithCurrency';
+import { EditableInput } from 'web-components/src/components/inputs/new/EditableInput/EditableInput';
+import { PrefinanceTag } from 'web-components/src/components/PrefinanceTag/PrefinanceTag';
+import { SupCurrencyAndAmount } from 'web-components/src/components/SupCurrencyAndAmount/SupCurrencyAndAmount';
+import Title from 'web-components/src/components/typography/new/Title';
+
+import {
+  OrderProps,
+  OrderSummaryProps,
+  PaymentMethod,
+} from './OrderSummaryCard.types';
+
+function OrderSummmaryRowHeader({
+  text,
+  className = '',
+}: {
+  text: string;
+  className?: string;
+}) {
+  return (
+    <Title
+      as="h3"
+      className={`text-grey-400 text-[11px] font-extrabold font-['Lato'] uppercase tracking-[1px] m-0 ${className}`}
+    >
+      {text}
+    </Title>
+  );
+}
+
+function OrderSummaryContent({
+  order,
+  currentBuyingStep,
+  paymentMethod,
+  onClickEditCard = () => {},
+}: {
+  order: OrderProps;
+  currentBuyingStep: number;
+  paymentMethod: PaymentMethod;
+  onClickEditCard?: () => void;
+}) {
+  const { projectName, currency, pricePerCredit, credits } = order;
+  const [creditsAmount, setCreditsAmount] = useState(credits);
+  return (
+    <div className="grid grid-cols-[75px_1fr] sm:grid-cols-[90px_1fr] max-w-full w-full pr-15 sm:px-[20px] pb-[30px] items-center sm:max-w-[330px]">
+      <Title
+        as="h5"
+        className="col-span-2 text-base font-black font-['Muli'] mt-0 sm:mt-[30px] mb-5 sm:mb-[15px]"
+      >
```

Comment:
you don't need to specify the font family in the className now that you're using `Title` component